EventListener/ExceptionListener.php

```diff
@@ -69,20 +79,34 @@ public function onKernelException($event)
                     break;
                 }
             }
+
             $this->dispatcher->removeListener(KernelEvents::CONTROLLER_ARGUMENTS, $controllerArgsListener);
         };
-        $this->dispatcher->addListener(KernelEvents::CONTROLLER_ARGUMENTS, $controllerArgsListener, -100);
+
+        $this->dispatcher->addListener(KernelEvents::CONTROLLER_ARGUMENTS, $controllerArgsListener);
 
         $this->exceptionListener->onKernelException($event);
     }
 
+    public function removeCspHeader(ResponseEvent $event): void
```

Comment:
I failed to see where this method is called. Can you verify if it's still necessary?

Reply:
this is forwards compatibility, see symfony/symfony master

Reply:
Oh! I see :) nvm.